pixeltable/func/expr_template_function.py

```diff
@@ -97,6 +96,9 @@ def exec(self, args: Sequence[Any], kwargs: dict[str, Any]) -> Any:
 
     @property
     def display_name(self) -> str:
+        if not self.self_name and isinstance(self.templates[0].expr, exprs.FunctionCall):
+            # In this common case, fall back on the display name of the underlying FunctionCall
```

Comment:
Could you spell out what this case is? I'm having trouble recognizing it.

Reply:
Done.